### ISSUE TITLE
Underscores in an url contained in the description of a submission get removed

### ISSUE THREAD

**Issue by pjdevries** [joomla/jed-issues#227]
**Describe the bug**
Underscores in an url conatined in the description of a submission get removed.

**To Reproduce**
Take a look at 'Edit Obix Class Extender'. Its description contains the following url: https://github.com/pjdevries/`
plg_system_classextender/blob/master/README.md. The url looks good in both editor and preview mode. On the JED page (https://extensions.joomla.org/extension/obix-class-extender/) the url is displayed like this:  https://github.com/pjdevries/plgsystemclassextender/blob/master/README.md.

In short: **plg_system_classextender** becomes **plgsystemclassextender**.

The actual description has been changed to prevent problems for visitors. The erroneous situation is represented in the screenshot below.

![image](https://user-images.githubusercontent.com/1171835/126166756-c9455aeb-2ff9-46e5-9223-85baa6b2dc86.png)

**Comment by Llewellynvdm** [comment]
@anibalsanchez this seems to be part of the JED that you are more familiar with. Can you tell us why this happens?

**Comment by anibalsanchez** [comment]
Not sure. It seems that the routine that cleans the markdown mistakenly changes the URLs in the description.

Please, note that URLs/links are not supported in the extension description. The only supported URLs/Links are the ones on the right sidebar.

**Comment by Llewellynvdm** [comment]
> Please, note that URLs/links are not supported in the extension description. The only supported URLs/Links are the ones on the right sidebar.

So this this means @pjdevries should just refrain from using URL's in the description. Thank you @anibalsanchez for the clarity on this. I will close this issue, @pjdevries let me know if you have any further questions. Please keep in mind that the JED is being rebuild as we speak and we will aim to handle this better in the future.
